### PR TITLE
Adds Clockwork Sect and Ratvarian Language

### PR DESCRIPTION
**Pull request by untrius**
## About The Pull Request

Adds the Clockwork religious sect, and gives them a little bit to do.

My intent here was to make a new set of things to do for the chaplain on the Shepard-class.  As I have written it, this is mostly RP fuel.

Features added:
1) Clockwork sect is now selectable from the Altar of the Gods.
-Using red alter sprite while a new one is being sprited
2) Metal/iron sheets can be sacrificed for 1 favor each.
-Could be extended to accept other metals
3) 60 favor can be traded for 30 brass via brass transmutation rite.
-Allows Shepard's crew to refit the ship in brass if they want
-Could add new rites to create more things from brass
4) Chaplain in clockwork sect knows the Ratvarian language and can teach it by blessing with their bible.
image
image

## Why It's Good For The Game

More stuff for people on the Shepard-class to do.

## Changelog
:cl:
add: Add Clockwork Sect
add: Re-add Ratvarian language
/:cl:

### CODE REVIEW

**Comment by untrius** [comment]
It's probably ready now, but I'm drafting it so I can stare at the code a little longer.

File: code/__HELPERS/ratvarian_language.dm [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+#define REVERSE_RATVAR_HYPHEN_TE_REPLACEMENT "$1"
+#define REVERSE_RATVAR_HYPHEN_ET_MATCH regex("-(\[eE]\[tT])","g")
+#define REVERSE_RATVAR_HYPHEN_ET_REPLACEMENT "$1"
+#define REVERSE_RATVAR_HYPHEN_GUA_MATCH regex("(\[gG]\[uU])-(\[aA])","g")
+#define REVERSE_RATVAR_HYPHEN_GUA_REPLACEMENT "$1$2"
+#define REVERSE_RATVAR_HYPHEN_OF_MATCH regex("-(\[oO]\[fF])","g")
+#define REVERSE_RATVAR_HYPHEN_OF_REPLACEMENT " $1"
+
+
+/proc/text2ratvar(text) //Takes english and applies ratvarian styling rules (and rot13) to it.
```
Comment (ZephyrTFA) [inline code/__HELPERS/ratvarian_language.dm:66]: Move the comment above and turn it into an autodoc comment

File: code/modules/religion/religion_sects.dm [hunk 2]
```diff
@@ -193,3 +193,37 @@
...
+/datum/religion_sect/clockwork
+	name = "Clockwork"
+	desc = "A sect oriented around gears and brass."
+	convert_opener = "Build for his honor, acolyte.<br>Bibles now teach the tongue of the Clockwork Justiciar. You can now sacrifice metal for favor."
+	alignment = ALIGNMENT_NEUT
+	desired_items = list(/obj/item/stack/sheet/metal)
+	rites_list = list(/datum/religion_rites/transmute_brass)
+	altar_icon_state = "convertaltar-red"
+
+/datum/religion_sect/clockwork/on_conversion(mob/living/L)
```
Comment (ZephyrTFA) [inline code/modules/religion/religion_sects.dm:207]:
```suggestion
/datum/religion_sect/clockwork/on_conversion(mob/living/acolyte)
```

**Review by ZephyrTFA — CHANGES_REQUESTED** [review]
Just a few minor things. 
Change your one letter var names to be more descriptive and move your comments above the thing they are commenting and turn them into autodoc comments

**Pull request merged**